Cargo.lock

```diff
@@ -2255,6 +2260,7 @@ dependencies = [
 "checksum parking_lot_core 0.2.14 (registry+https://github.com/rust-lang/crates.io-index)" = "4db1a8ccf734a7bce794cc19b3df06ed87ab2f3907036b693c68f56b4d4537fa"
 "checksum parking_lot_core 0.4.0 (registry+https://github.com/rust-lang/crates.io-index)" = "94c8c7923936b28d546dfd14d4472eaf34c99b14e1c973a32b3e6d4eb04298c9"
 "checksum percent-encoding 1.0.1 (registry+https://github.com/rust-lang/crates.io-index)" = "31010dd2e1ac33d5b46a5b413495239882813e0369f8ed8a5e266f173602f831"
```

Comment:
You'll have to add the dupe to the servo-tidy.toml file (or even better, fix the dupe :D)

Reply:
Fixing the duplication requires on of https://github.com/rust-windowing/winit/pull/1076 and winit v0.19.2 or crates.io, or winit v0.20.0 to be finished and published on crates.io. (Assuming we’d rather not use an alpha version.) Let’s wait and see if the maintainers are open to making 0.19.2.

Reply:
Ok, so…

```rust
    Updating crates.io index
      Adding lock_api v0.2.0
      Adding parking_lot v0.8.0
      Adding parking_lot_core v0.5.0
    Removing percent-encoding v1.0.1
      Adding scopeguard v1.0.0
    Updating winit v0.19.1 -> v0.19.2
```

Reply:
```rust
./Cargo.lock:1: duplicate versions for package `lock_api`
	The following packages depend on version 0.1.3 from 'crates.io':
		parking_lot
		parking_lot
	The following packages depend on version 0.2.0 from 'crates.io':
		parking_lot
./Cargo.lock:1: duplicate versions for package `scopeguard`
	The following packages depend on version 0.3.3 from 'crates.io':
		crossbeam-epoch
		crossbeam-epoch
		lock_api
	The following packages depend on version 1.0.0 from 'crates.io':
		lock_api